System/Database/DatabaseInterface.py

```diff
@@ -4,6 +4,7 @@
 
 #import mysql.connector
 import pymysql
+import encryption
```

Comment:
Did you mean "CLI.Encryption"?

If the encryption module spans across the CLI and the Database, perhaps we should make a crypto folder and move all cryptography files there.